### ISSUE TITLE
Better landmarks in our apps

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1925]
Testing in an app I saw that we have three navigation landmarks. Some with cryptic names. Landmarks are used by screen readers and should clearly define the different regions of the page. We should clean up how we handle landmarks to make our apps more accessible. 

![1](https://github.com/Altinn/app-frontend-react/assets/33222679/74b0f03d-a3bc-4fe0-a5bd-9015557bdcea)

### Suggestion
![Skjermbilde 2024-04-04 kl  09 23 03](https://github.com/Altinn/app-frontend-react/assets/33222679/bb309e2c-3c54-4b5d-935b-196fb7ec86b0)
* `<header>` keep as is. 
  * Move language picker into header. Might be fixed by #1431 
* `<nav>` as is but create a `aria-label`
  * We need to find out if the page navigation should be contained in the same navigation or a sepparate one. related to: https://github.com/Altinn/app-frontend-react/issues/97 
* `<main>`Keep as is. But move the page navigation out of main.
* `<footer>`Keep as is. 
* The only thing missing is the service title. I dont think this should be included in main because it is the same on all pages. But it should be inside a landmark.
* Maybe not use navigation in the `aria-label` as it is implisit in the landmark

```[tasklist]
### Tasks
- [ ] Define landmarks used today
- [ ] What needs to be changed
- [ ] Look into `form` landmark
```

**Comment by Janingetjosvoll** [comment]
Har dere noe estimat for når denne blir sett på?
